File: pgm/ABC_Dart/ABC_421_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final int N = int.parse(stdin.readLineSync()!);
  List<String> S = [];
  for (int i = 0; i < N; i++) {
    S.add(stdin.readLineSync()!);
  }
  final (int X, String Y) = ((arr) =>
      (int.parse(arr[0]), arr[1]))(stdin.readLineSync()!.split(' '));
  const String Yes = 'Yes';
  const String No = 'No';
  const int one = 1;
  if (S[X - one] == Y) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}

File: pgm/ABC_C#/ABC_421_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        string[] S = new string[N];
        const string Yes = "Yes";
        const string No = "No";
        const char space = ' ';
        const int one = 1;
        const int zero = 0;
        for (int i = zero; i < N; i++)
        {
            S[i] = Console.ReadLine();
        }
        string[] arr = Console.ReadLine().Split(space);
        int X = int.Parse(arr[zero]);
        string Y = arr[one];
        Console.WriteLine(S[X - one] == Y ? Yes : No);
        Environment.Exit(0);
    }
}